docs/build/configure/_index.md

```diff
@@ -239,242 +240,29 @@ For example, if you have multiple machines with the same motor hardware, wired t
 To configure a fragment, click the **+** icon next to your {{< glossary_tooltip term_id="part" text="machine part" >}} in the left-hand menu of the **CONFIGURE** tab and select **Insert fragment**.
 See [Use Fragments to Configure a Fleet](/fleet/configure-a-fleet/) for more information on creating and deploying fragments.
 
-## Network
+### Configure the frame system
 
-Expand a part's configuration card to open its network configuration interface:
-
-{{<imgproc src="/build/configure/network.png" resize="x400" style="max-width: 300px" declaredimensions=true alt="The network configuration interface on a part card of a machine.">}}
+The frame system holds reference frame information for the relative position of components in space.
 
-You can configure the address `viam-server` binds to for accepting connections.
-By default, `viam-server` binds to `0.0.0.0:8080` when managed by the Viam app or when authentication and TLS are enabled.
-You can also set the [heartbeat](/build/program/apis/sessions/#heartbeats) window.
+Clicking on the **Frame** mode to visualize and configure the relative positions of components.
+Find more information in the [frame system documentation](/mobility/frame-system/).
 
-## Webhooks
+### Add a webhook
```

Comment:
I realize we want to be active, and to be consistent, but I feel this heading could frustrate users if they come here expecting how-to content and have to follow the link to find it. Since this heading just contains an introduction to the concept, not how to add one, can it stay as it was?
```suggestion
### Webhooks
```
Same comment on components, services and frame system (though out of diff)

Reply:
I think that's fair, will change